app/models/tribunal_case.rb

```diff
@@ -61,4 +65,28 @@ def started_by_taxpayer?
   def started_by_representative?
     user_type.eql?(UserType::REPRESENTATIVE)
   end
+
+  private
+
+  def sanitize
+    self.class.columns.each do |col|
+      # Skip uuids, integers, datetimes, et al.
+      next unless [:string, :text].include?(col.type)
+      value = public_send(col.name)
+      # Do not attempt to sanitize nils or value object.
+      if value.instance_of?(String)
+        public_send("#{col.name}=", sanitizer(value))
+      end
+    end
+  end
+
+  def sanitizer(value)
+    CGI.escapeHTML(Sanitize.fragment(value, Sanitize::Config::RESTRICTED)).
+      gsub('*', '&#42;').
+      gsub('=', '&#61;').
+      gsub('-', '&dash;').
+      gsub('%', '&#37;').
+      gsub(/drop\s+table/i, '').
+      gsub(/insert\s+into/i, '')
```

Comment:
I just thought of something else - are ruby regexes multi-line, by default? Would this catch;

drop
table foo

for instance?